src/web/topic/components/TopicWorkspace/MoreActionsDrawer.tsx

```diff
@@ -141,6 +149,44 @@ export const MoreActionsDrawer = ({
   const fillNodesWithColor = useFillNodesWithColor();
   const expandDetailsTabs = useExpandDetailsTabs();
 
+  const [resetDialogOpen, setResetDialogOpen] = useState(false);
+
+  const resetComments = (
```

Comment:
```suggestion
  const resetDialog = (
```

This variable is the dialog component - `resetComments` is a little misleading, as if it's a method rather than dialog component

Reply:
The component name "resetComments" has been changed to "resetDialog"